AVIARY DIM [-1 -1  0  1  1  1]
Attempting to open: C:\Files\Egyetem\Szakdolgozat\RL\Sol/resources
[INFO] BaseAviary.__init__() loaded parameters from the drone's .urdf:
[INFO] m 0.027000, L 0.039700,
[INFO] ixx 0.000014, iyy 0.000014, izz 0.000022,
[INFO] kf 0.000000, km 0.000000,
[INFO] t2w 2.250000, max_speed_kmh 30.000000,
[INFO] gnd_eff_coeff 11.368590, prop_radius 0.023135,
[INFO] drag_xy_coeff 0.000001, drag_z_coeff 0.000001,
[INFO] dw_coeff_1 2267.180000, dw_coeff_2 0.160000, dw_coeff_3 -0.110000
Traceback (most recent call last):
  File "C:\Files\Egyetem\Szakdolgozat\RL\Sol\Model\pybullet_drone_simulator.py", line 658, in <module>
    sim.run_full(args)
  File "C:\Files\Egyetem\Szakdolgozat\RL\Sol\Model\pybullet_drone_simulator.py", line 308, in run_full
    custom_policy = CustomActorCriticPolicy(train_env.observation_space, train_env.action_space,
  File "C:\Files\Egyetem\Szakdolgozat\RL\Sol\Model\SBActorCritic.py", line 61, in __init__
    super(CustomActorCriticPolicy, self).__init__(observation_space, action_space,
  File "C:\Users\xx4qw\anaconda3\envs\CondaDrone\lib\site-packages\stable_baselines3\common\policies.py", line 533, in __init__
    self._build(lr_schedule)
  File "C:\Users\xx4qw\anaconda3\envs\CondaDrone\lib\site-packages\stable_baselines3\common\policies.py", line 590, in _build
    self._build_mlp_extractor()
  File "C:\Files\Egyetem\Szakdolgozat\RL\Sol\Model\SBActorCritic.py", line 102, in _build_mlp_extractor
    self.mlp_extractor = CustomFeatureExtractor(self.observation_space, self.net_arch, self.activation_fn)
  File "C:\Files\Egyetem\Szakdolgozat\RL\Sol\Model\SBActorCritic.py", line 30, in __init__
    super(CustomFeatureExtractor, self).__init__(observation_space,
  File "C:\Users\xx4qw\anaconda3\envs\CondaDrone\lib\site-packages\stable_baselines3\common\torch_layers.py", line 23, in __init__
    assert features_dim > 0
TypeError: '>' not supported between instances of 'list' and 'int'